Yafc.Parser/Data/FactorioDataDeserializer_Entity.cs

```diff
@@ -435,7 +435,7 @@ internal partial class FactorioDataDeserializer {
             entity.size = table.Get("selection_box", out LuaTable? box) ? GetSize(box) : 3;
 
             _ = table.Get("energy_source", out LuaTable? energySource);
-            if (factorioType != "generator" && factorioType != "solar-panel" && factorioType != "accumulator" && factorioType != "burner-generator" && factorioType != "offshore-pump" && energySource != null) {
+            if (factorioType != "generator" && factorioType != "solar-panel" && factorioType != "accumulator" && factorioType != "burner-generator" && factorioType != "offshore-pump" && factorioType != "electric-energy-interface" && energySource != null) {
```

Comment:
We could make a list and check if the type is not in the list? This way it might clear up a little (related to @shpaass' comment)